src/jackknife.jl

```diff
@@ -127,7 +127,7 @@ julia> jackknife_variance(:api00, weightedmean, rstrat)
 # Reference
 pg 380-382, Section 9.3.2 Jackknife - Sharon Lohr, Sampling Design and Analysis (2010)
 """
-function jackknife_variance(x::Symbol, func::Function, design::ReplicateDesign)
+function jackknife_variance(x::Symbol, func::Function, design::ReplicateDesign{JackknifeReplicates})
```

Comment:
It should just be called variance. There should be 2 dispatches. 
1. `variance(x::Symbol, func::Function, design::ReplicateDesign{JackknifeReplicates})`
2.  `variance(x::Symbol, func::Function, design::ReplicateDesign{BootstrapReplicates})`

Reply:
Okay, but we don't have a `variance` method for `BootstrapReplicates` method yet, do we?

Reply:
Also, should I rewrite the mean function using multiple dispatch now?

Reply:
We should rewrite the functions now.
And we need a variance function for `BootstrapReplicates`. It's pretty simple, it's just the simple variable of the estimates.

Reply:
> We should rewrite the functions now. And we need a variance function for `BootstrapReplicates`. It's pretty simple, it's just the simple variable of the estimates.

Okay. Also, just to confirm: haven't we already implemented the variance function for `BootstrapReplicates` in the `mean` function in line 48 of this file?: https://github.com/xKDR/Survey.jl/blob/main/src/mean.jl

So we should probably get rid of this `mean` function right?

Reply:
Yes, we should get rid of it, and
```
function variance(x::Symbol, func::Function, design::ReplicateDesign{BootstrapReplicates})
...
variance = sum((θ̂t .- θ̂) .^ 2) / design.replicates
end
```

Reply:
Done